defaults/main.yml

```diff
@@ -62,6 +62,12 @@ he_vm_mac_addr: null
 he_remove_appliance_rpm: true
 he_pki_renew_on_restore: false
 
+# If you decide to change these variables,
+# make sure they have the subnet mask
```

Comment:
in default.xml we have:
  <ip address='192.168.122.1' netmask='255.255.255.0'>
but we are not editing the netmask attribute (and honestly I'm neither that sure it's worth).

Reply:
have what ?